packages/expo/src/ScreenOrientation.ts

```diff
@@ -1,6 +1,16 @@
 import { NativeModules } from 'react-native';
 
-const { ExponentScreenOrientation } = NativeModules;
+const {
+  ExponentScreenOrientation = {
```

Comment:
Move implementation into a separate file and add TS type annotations. We're going to add more to this with the new version of the ScreenOrientation module.